src/Helper/Routing/Url/Transformer.php

```diff
@@ -60,7 +61,7 @@ public function generate(array $match, ?string $locale = null)
                 throw new \Exception('missing template');
             }
 
-            return $this->generator->prependBaseUrl($emsLink, $template);
+            return $this->generator->prependBaseUrl($template);
```

Comment:
What's happened to the emsLink?

Reply:
In the past the generator needed the emsLink for determing the contentType and then check the relativePaths array. But this array we don't use anymore. Or github website-skeleton does not even support this configuration anymore ...